lib/board.ex

```diff
@@ -0,0 +1,62 @@
+defmodule Board do
+
+  defstruct [:size, :cells, :marks]
+
+  def create_cells(size) do
+    Enum.to_list(1..size * size)
+  end
+
+  def place_mark(board, position, mark) do
+    %{board | cells: List.replace_at(board.cells, position - 1, mark)}
+  end
+
+  def rows(board) do
+    Enum.chunk_every(board.cells, board.size)
+  end
+
+  def columns(board) do
+    rows(board)
```

Comment:
board
|> rows()

So that you have the continuuing |> syntax 😄

Reply:
🆒